colossalai/gemini/tensorful_state.py

```diff
@@ -0,0 +1,172 @@
+from enum import Enum
```

Comment:
the name should be stateful_tensor?

Reply:
Yes. But It is called tensorful_state.py originally, I just move it to /gemini.

Reply:
I have corrected its name.